app.go

```diff
@@ -341,9 +355,13 @@ func (app *App) Listen(address interface{}, tlsconfig ...*tls.Config) error {
 	if len(tlsconfig) > 0 {
 		ln = tls.NewListener(ln, tlsconfig[0])
 	}
-	// Print listening message
+	// Print startup message
 	if !app.Settings.DisableStartupMessage && !utils.GetArgument("-child") {
-		fmt.Printf("        _______ __\n  ____ / ____(_) /_  ___  _____\n_____ / /_  / / __ \\/ _ \\/ ___/\n  __ / __/ / / /_/ /  __/ /\n    /_/   /_/_.___/\\___/_/ v%s\n", Version)
+		if !app.Settings.HideStartupBanner {
+			if !app.Settings.HideStartupBanner {
+				fmt.Print(app.Settings.StartupBanner)
```

Comment:
```suggestion
				fmt.Println(app.Settings.StartupBanner)
```

Reply:
@renanbastos93 The thing is that previously the StartupBanner was written directly with `fmt.Printf` (you can see it in removed line 346), because the newline is already included in the banner text, so use `fmt.Println` would result in an extra line that wasn't there before.
So, if this change is applied, we should then remove the final line in the `fiberASCIIBanner`, to keep it the same way it has been rendered since always it seems.

Reply:
sure